app/src/ui/resizable/resizable.tsx

```diff
@@ -150,6 +169,24 @@ export class Resizable extends React.Component<IResizableProps> {
     this.resizeContainer = ref
   }
 
+  private getResizePercentage() {
+    const minWidth = this.props.minimumWidth ?? 0
+    const maxWidth = this.props.maximumWidth ?? DefaultMaxWidth
+    return Math.round(
+      ((this.getCurrentWidth() - minWidth) / (maxWidth - minWidth)) * 100
+    )
+  }
+
+  private updateResizeMessage(direction: ResizeDirection) {
+    const directionMessage =
+      direction === ResizeDirection.Increase ? 'increased' : 'decreased'
+    this.setState({
+      resizeMessage: `${
+        this.props.description
+      } width ${directionMessage}. Set to ${this.getResizePercentage()}%`,
+    })
```

Comment:
More accurately "Set to x% of available width" since available width is determined by app window.. which will change.. thus if you set it to 50% and then made the window wider it may now be 30%.. But... I don't think that "of available width" communicates that and just adds clutter to an already long message.